path_login.go

```diff
@@ -222,129 +221,98 @@ func (b *kubeAuthBackend) aliasLookahead(ctx context.Context, req *logical.Reque
 	}, nil
 }
 
+// parseAndVerifySignature parses the JWT token validating the signature against
+// any of the keys passed in.
+func parseAndVerifySignature(token string, keys ...interface{}) (*jwt.Token, error) {
+	for _, k := range keys {
+		result, err := jwt.Parse(token, func(t *jwt.Token) (interface{}, error) {
+			return k, nil
+		})
+		if result != nil && err == nil {
+			return result, nil
+		}
+		if err != nil && strings.ToLower(err.Error()) == "token is expired" {
```

Comment:
Sadly, no, because of the weird capitalization issue.

`jwt.ErrTokenExpired`'s message is `token is expired`, but the error message we see is `Token is expired`